docs/develop/specify-office-hosts-and-api-requirements.md

```diff
@@ -192,6 +208,42 @@ if (Office.context.document.setSelectedDataAsync)
 }
 ```
 
+### Specifying requirements in a VersionOverrides element
+
+The [VersionOverrides](../reference/manifest/versionoverrides.md) element was added to the manifest schema primarily, but not exclusively, to support features that must be available immediately after an add-in is installed, such as add-in commands (custom ribbon buttons and menus). Office must know about these features when it parses the add-in manifest. 
+
+When your add-in uses one of these features, but the add-in is valuable, and should be installable, even on Office versions that don't support the feature, you can identify the feature using a [Requirements](../reference/manifest/requirements.md) element (and its child [Sets](../reference/manifest/sets.md) and [Methods](../reference/manifest/methods.md) elements) *as a child of the **VersionOverrides** element itself instead of in the base manifest (that is, as a child of the `OfficeApp` element*). The effect of doing this is that Office will allow the the add-in to be installed, but Office will ignore certain of the child elements of the **VersionOverrides** element on Office versions where the feature is not supported. 
```

Comment:
```suggestion
Suppose your add-in uses one of these features, but the add-in is valuable, and should be installable, even on Office versions that don't support the feature. In this scenario, identify the feature using a [Requirements](../reference/manifest/requirements.md) element (and its child [Sets](../reference/manifest/sets.md) and [Methods](../reference/manifest/methods.md) elements) that you include as a child of the **VersionOverrides** element itself instead of in the base manifest (that is, as a child of the `OfficeApp` element). The effect of doing this is that Office will allow the add-in to be installed, but Office will ignore certain of the child elements of the **VersionOverrides** element on Office versions where the feature is not supported. 
```